overlord/snapstate/backend/link.go

```diff
@@ -184,6 +185,25 @@ func (b Backend) LinkSnap(info *snap.Info, dev snap.Device, linkCtx LinkContext,
 	return rebootInfo, nil
 }
 
+func componentLinkPath(mountDir string) (string, error) {
+	// The symlink to current is the mount with the stripped revision
+	revStart := strings.LastIndex(mountDir, "_")
+	if revStart == -1 {
+		return "", fmt.Errorf("internal error: no revision in %q", mountDir)
+	}
+	return mountDir[0:revStart], nil
```

Comment:
were `/snap/<name>/components/<comp>/current` and `/snap/<name>/components/<comp>/<rev>` maybe considered as paths we could use?

Reply:
Yes, I was a bit between the two options, what I implemented is shorter and matches the path defined in the spec, and seems to me cleaner, but using `.../current` makes this consistent with snaps.

Reply:
FTR the agreement has been:
* Symlinks paths will still be `/snap/<snap>/components/<snap_revision>/<component_name>`
* Components will be mounted on `/snap/<snap>/components/mnt/<comp_name>/<comp_revno>`